File: szavazas-core/src/main/java/hu/kdea/szavazas/ballotprocessor/qr/QrResult.java
package hu.kdea.szavazas.ballotprocessor.qr;

import hu.kdea.szavazas.ballotprocessor.common.Rect;

public final class QrResult {
    private final String raw;
    private final int numSupport;
    private final int numCandidates;
    private final Rect boundingBox;

    public QrResult(String raw, int numSupport, int numCandidates, Rect boundingBox) {
        this.raw = raw;
        this.numSupport = numSupport;
        this.numCandidates = numCandidates;
        this.boundingBox = boundingBox;
    }

    public String getRaw() {
        return raw;
    }

    public int getNumSupport() {
        return numSupport;
    }

    public int getNumCandidates() {
        return numCandidates;
    }

    public Rect getBoundingBox() {
        return boundingBox;
    }
}

File: test-utils/src/main/java/hu/kdea/szavazas/ballotprocessor/qr/QRResult.kt
// QRResult.kt
package hu.kdea.szavazas.ballotprocessor.qr

import hu.kdea.szavazas.ballotprocessor.common.Rect

data class QrResult(
    val raw: String,
    val numSupport: Int,
    val numCandidates: Int,
    val boundingBox: Rect   // now our custom Rect
)